packages/swatch/src/SwatchGroup.ts

```diff
@@ -191,7 +192,7 @@ export class SwatchGroup extends SizedMixin(SpectrumElement, {
         }
         if (
             changes.has('size') &&
-            (this.size || typeof changes.get('size') !== 'undefined')
+            (this.size !== 'm' || typeof changes.get('size') !== 'undefined')
```

Comment:
this can fail if size has been given as 'M' and not 'm'?
`this.size.toLowerCase() `

Reply:
Super valuable point and why we validate and normalize these values in the setter: https://github.com/adobe/spectrum-web-components/blob/main/tools/base/src/sizedMixin.ts#L55-L72